roles/server/templates/nginx_site.j2

```diff
@@ -1,26 +1,24 @@
 server {
     listen 80;
-    listen [::]:80;
 
-    server_name korobi.dev;
+    server_name {{ host }};
 
-    return 301 https://korobi.dev$request_uri;
+    return 302 https://$host$request_uri;
```

Comment:
Should be a 301, we have no plans to not redirect to the HTTPS'd version of the site.